libs/langchain-community/src/vectorstores/weaviate.ts

```diff
@@ -88,7 +88,7 @@ interface ResultRow {
  */
 export interface WeaviateFilter {
   distance?: number;
-  where: WhereFilter;
+  where?: WhereFilter;
```

Comment:
This will be a breaking change to anyone using this and not already passing `where`